src/tools/debugger/session/Breakpoints.java

```diff
@@ -195,6 +212,70 @@ public Breakpoint getAsyncMessageRcvBreakpoint(final URI sourceUri,
     return bp;
   }
 
+  public synchronized void addReceiverBreakpoint(final URI sourceUri, final int startLine,
+      final int startColumn, final int charLength) {
+    SectionBreakpoint bId = new SectionBreakpoint(sourceUri, startLine, startColumn, charLength);
```

Comment:
Yeah, no, I prefer to have the concrete type here, especially since it goes into a map with breakpoints that is only supposed to contain receiver breakpoints. And, it makes the code using that map simpler.